go.mod

```diff
@@ -1,18 +1,19 @@
 module github.com/cometbft/cometbft-db
 
-go 1.19
+go 1.20
 
 require (
-	github.com/cockroachdb/pebble v0.0.0-20230315005856-dcb60b9212f9
+	// used by cockroach v23.1.10
+	github.com/cockroachdb/pebble v0.0.0-20230807182518-7bcdd55ef1e3
 	github.com/dgraph-io/badger/v2 v2.2007.4
 	github.com/gogo/protobuf v1.3.2
 	github.com/google/btree v1.1.2
 	github.com/jmhodges/levigo v1.0.0
-	github.com/linxGnu/grocksdb v1.7.15
-	github.com/stretchr/testify v1.8.2
+	github.com/linxGnu/grocksdb v1.8.0
```

Comment:
https://github.com/linxGnu/grocksdb/compare/v1.7.16...v1.8.0

Reply:
1.8.0 is to support rocksdb v8.5.3